libretro_core_options.h

```diff
@@ -630,7 +630,7 @@ struct retro_core_option_v2_definition option_defs_us[] = {
       BEETLE_OPT(memcard_left_index),
       "Memory Card Left Index",
       NULL,
-      "Changes the memory card currently loaded in the left slot. This option will only work if Memory Card 0 method is set to Mednafen. The default card is index 0.",
+      "Change the memory card currently loaded in the left slot. This option will only work if Memory Card 0 method is set to Mednafen. The default card is index 0.",
```

Comment:
Lowercase the second "Memory Card" to unify all cases. Seems that Sony didn't trademark the words "Memory Card", so they don't require mandatory uppercasing. Also, "Mednafen" should be quoted to indicate that it is an option.